.github/workflows/testBuild.yml

```diff
@@ -71,6 +71,9 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha || needs.getBranchRef.outputs.REF }}
 
+      - name: Add PULL_REQUEST_NUMBER to .env.staging file
```

Comment:
I can appreciate the simplicity of this but I wonder if creating a `.env.adhoc` is more clear?

Reply:
I took that into consideration, but quoting Rory from [this discussion](https://github.com/Expensify/App/issues/13993#issuecomment-1378062235) :)

> we could create a separate environment file for ad-hoc builds. The downside is that we'd be introducing a new environment file that we have to maintain, and that environment file would be different from the staging environment. In general I think we want these ad-hoc builds to be as similar to staging builds as possible.

Reply:
@roryabraham - Curious for your thoughts here now that we have a PR up. My sense is adhoc is not a ton to maintain and it's more explicit as we now have two or three different terms to describe "adhoc" "pull request" "internal testing", etc.